controllers/BooksShowController.js

```diff
@@ -0,0 +1,52 @@
+angular
+  .module('booksApp')
+  .controller('BooksShowController', BooksShowController);
+
+var db = "https://super-crud.herokuapp.com/books"
+
+BooksShowController.$inject = ['$http', '$routeParams', '$location'];
+function BooksShowController(   $http,   $routeParams,  $location){
+
+  var vm = this;
+
+  vm.book = {};
+
+  vm.editedBook = {};
+
+  vm.editing = function(){
+    vm.editedBook = Object.assign({}, vm.book);
+  }
+
+  vm.editBook = function(){
+    $http({
+      method: 'PUT',
+      url: `${db}/${$routeParams.id}`,
```

Comment:
**reviewer disagreement whooo**
I agree that you could just add the slash to your url declaration and use +, but I also like your use of template literals. /shrug